terraform/services/zscaler-security-groups/main.tf

```diff
@@ -0,0 +1,23 @@
+resource "aws_security_group" "zscaler_traffic_allow_public" {
+  name        = "${var.app}-${var.env}-allow-zscaler-public"
+  description = "Allow internet zscaler traffic public"
+  vpc_id      = var.vpc_id
+  ingress {
```

Comment:
The way security groups are managed in the AWS terraform provider has recently been updated. These CIDRs should be moved to aws_vpc_security_group_ingress_rule resources. See the first note at https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/security_group.html:

> The [aws_vpc_security_group_egress_rule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_security_group_egress_rule) and [aws_vpc_security_group_ingress_rule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_security_group_ingress_rule) resources have been added to address these limitations and should be used for all new security group rules.

Reply:
Please update to use aws_vpc_security_group_ingress_rule resources as noted in the comment above.